web/docs/language/features.md

```diff
@@ -1236,10 +1311,10 @@ If you require custom configuration setup or user entity field assignment, you c
 
 NOTE: The same UI helpers apply as for Google. Please [see here](features#google-ui-helpers) for more.
 
-### `externalAuthEntity`
-Anytime an authentication method is used that relies on an external authorization provider, for example, Google, we require an `externalAuthEntity` specified in `auth` that contains at least the following highlighted fields:
+#### `externalAuthEntity`
+Anytime an authentication method is used that relies on an external authorization provider, for example, Google, we require an `externalAuthEntity` specified in `auth`, in addition to the `userEntity`, that contains at least the following highlighted fields:
 
```

Comment:
Note for future: I've noticed that `features.md` is a very long file. We should break it up into multiple files, so it's easier to write and it's easier to format.

```
languages/ 
-- features.md
-- social-auth.md
-- social-auth/
---- google.md
---- github.md
```

Reply:
Yeah I noticed this also. I think we need to do a big clean-up of our docs. Looking at the RedwoodJS docs makes that very apparent

Reply:
Yup, we certainly do need to organize it better! We need to split it + we could use nicer UI.